Util/exact_riemann/riemann_rarefaction.H

```diff
@@ -0,0 +1,432 @@
+#ifndef RIEMANN_RAREFACTION_H
+#define RIEMANN_RAREFACTION_H
+
+#include <utility>
+
+#include <AMReX_REAL.H>
+
+#include <eos_type.H>
+#include <eos.H>
+
+
```

Comment:
Because we assume only rarefractions in this header file, we plan to integrate the Riemann invariants $\mathbf{L}^{(+,-)} \cdot \mathbf{dq} = 0$, and  $\mathbf{L}^{(0)} \cdot \mathbf{dq} = 0$. The variable `iwave` represents the characteristic wave, 1 for the left wave, 2 for the contact wave and 3 for the right one.

Reply:
are you suggesting to add this as a comment?